nix/wire-server.nix

```diff
@@ -518,7 +518,7 @@ in
       pkgs.netcat
       pkgs.niv
       pkgs.haskellPackages.apply-refact
-      (pkgs.python310.withPackages
+      (pkgs.python3.withPackages
```

Comment:
Have you tried building the docs? I remember there was something broken.

Reply:
this is not the python that is used by the docs, it's not used by anything really. mainly a convience if you want to write python scripts in hack/bin